node.go

```diff
@@ -204,9 +210,11 @@ func (n *ProtoNode) Copy() ipld.Node {
 	return nnode
 }
 
-// RawData returns the protobuf-encoded version of the node.
 func (n *ProtoNode) RawData() []byte {
-	out, _ := n.EncodeProtobuf(false)
+	out, err := n.EncodeProtobuf(false)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
It looks like we previously just ignored the errors. While I'd generally rather panic than silently ignore errors I'd like to know what can cause an error here and if any of it is user/network (rather than developer) issues.

Reply:
I'm going with "that's what it does everywhere else in the code and this brings it in line"? But you'll have to ask @mvdan cause he's on the blame. But I don't feel attached.

Reply:
An error (and thus panic) can happen if the IPLD node is malformed, or if the checksum for the CID fails. Both should basically happen never, unless something went terribly wrong, like memory corruption or an internal bug.

Reply:
ok, as long as it isn't I/O or user based this is probably fine.